spock-specs/specs.gradle

```diff
@@ -49,6 +49,9 @@ task groovyConsole(type: JavaExec) {
 
 compileTestGroovy {
   def compileTestGroovyExec = "$buildDir/jacoco/compileTestGroovy.exec"
-  groovyOptions.fork jvmArgs: ["-javaagent:${configurations.jacocoAgentRuntime.asPath}=destfile=$compileTestGroovyExec,dumponexit=false,append=false,jmx=true,includes=org.spockframework.*".toString()]
+  groovyOptions.fork jvmArgs: [
+    "-javaagent:${configurations.jacocoAgentRuntime.asPath}=destfile=$compileTestGroovyExec,dumponexit=false,append=false,jmx=true,includes=org.spockframework.*".toString(),
+    "-DJacocoAstDumpTrigger=true"
```

Comment:
This does not match `.name`, either use `.simpleName` or set `org.spockframework.specs.jacoco.JacocoAstDumpTrigger` here.